cartographer/pose_graph/constraint/relative_pose_constraint_2d.cc

```diff
@@ -70,7 +70,7 @@ void RelativePoseConstraint2D::AddToOptimizer(Nodes* nodes,
   if (second_node->constant()) {
     problem->SetParameterBlockConstant(second_pose->data());
   }
-  problem->AddResidualBlock(ceres_cost_.get(), nullptr /* loss function */,
+  problem->AddResidualBlock(&ceres_cost_, nullptr /* loss function */,
```

Comment:
nah, does not work:

relative_pose_constraint_2d.cc:73:29: error: invalid conversion from ‘const ceres::CostFunction*’ to ‘ceres::CostFunction*’ [-fpermissive]
   problem->AddResidualBlock(&ceres_cost_, nullptr /* loss function */,